python/training/export_nl_graph.py

```
@@ -0,0 +1,584 @@
+        # Summary stats
+        total_nodes = sum(data[nt].x.size(0) for nt in data.node_types if hasattr(data[nt], 'x') and data[nt].x is not None)
+        total_edges = sum(data[et].edge_index.size(1) for et in data.edge_types if hasattr(data[et], 'edge_index'))
+        print(f"  Nodes:     {total_nodes} across {len(data.node_types)} types")
+        print(f"  Edges:     {total_edges} across {len(data.edge_types)} triplets")
+
+
+# ---------------------------------------------------------------------------
+# Verification
+# ---------------------------------------------------------------------------
+
+def verify(output: Path, verbose: bool = True) -> bool:
```

Ruff (0.15.2)

[warning] Unused function argument: `verbose`

(ARG001)